src/style.css

```diff
@@ -1 +1,76 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir.ttc") format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-book;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir-heavy;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 900;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  line-height: normal;
+  font-style: normal;
+  color: #3d4e61;
+  margin: 20px 8px;
+  background: #fff;
+}
+
+.input {
+  font-family: Avenir-book, sans-serif;
```

Comment:
```suggestion
```

Reply:
changed, the error is the same
![Screenshot_1](https://github.com/mate-academy/layout_search-bar-airbnb/assets/149514552/0fcbd921-d4d9-465a-82a6-13b5db094fc8)
![Screenshot_2](https://github.com/mate-academy/layout_search-bar-airbnb/assets/149514552/0228c507-ddd8-4ce8-a89e-1a6efc96adea)
![Screenshot_3](https://github.com/mate-academy/layout_search-bar-airbnb/assets/149514552/fad98959-d20c-4f1c-b05d-48b57604ef02)

Reply:
![Screenshot_1](https://github.com/mate-academy/layout_search-bar-airbnb/assets/149514552/2c65a47d-b888-4b4d-a610-a74a7840e26f)